gobblin-runtime/src/main/java/org/apache/gobblin/runtime/troubleshooter/JobIssueEventHandler.java

```diff
@@ -50,6 +51,7 @@ public class JobIssueEventHandler {
   private static final Logger issueLogger =
       LoggerFactory.getLogger("org.apache.gobblin.runtime.troubleshooter.JobIssueLogger");
 
+  @Getter
```

Comment:
any expectations to document here, such as only read-only use... or doesn't matter?

Reply:
With your suggestion to move the class as parameter we no longer need this :)